dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/logservice/ApplicationLoggerSettings.java

```diff
@@ -83,10 +98,12 @@ public PatternLayoutEncoder updateLogBackSettings(final String pattern) {
         if (logSettings.getDebugLogging()) {
             rootLogger.setLevel(Level.DEBUG);
         } else {
-            rootLogger.setLevel(Level.ERROR);
+            rootLogger.setLevel(Level.INFO);
```

Comment:
This is probably not something that we need to worry about with INFO.  It will help us with debugging in the field if it is set to this.